egs/iam/s5/local/chain/run_cnn_1a.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash
+
+# steps/info/chain_dir_info.pl exp/chainfsf4/cnn1a_1/
+# exp/chainfsf4/cnn1a_1/: num-iters=21 nj=2..4 num-params=4.4M dim=40->380 combine=-0.033->-0.025 xent:train/valid[13,20,final]=(-1.07,-1.31,-0.560/-1.30,-1.70,-0.978) logprob:train/valid[13,20,final]=(-0.064,-0.119,-0.011/-0.115,-0.208,-0.096)
+
+# head exp/chainfsf4/cnn1a_1/decode_test/scoring_kaldi/best_wer
+#%WER 18.34 [ 3231 / 17616, 348 ins, 693 del, 2190 sub ] exp/chainfsf4/cnn1a_1/decode_test/wer_8_1.0
```

Comment:
Please fix the directory names in these results (i.e. chain chainfsf4 to chain and cnn1a_1 to cnn1a)
BTW, we will later need to update them when you get the LM working so you can do it then.